src/display/editor/annotation_editor_layer.js

```diff
@@ -809,8 +812,18 @@ class AnnotationEditorLayer {
   }
 
   createAnnotationNode() {
-    const { event, data } = this.#highlightSelectionParams;
-    this.#createAndAddNewEditor(event, false, data);
+    const editorType = this.#currentEditorType;
+    if (editorType) {
+      const { event, data } = this.#highlightSelectionParams;
+      this.#createAndAddNewEditor(event, false, data);
+      this.#highlightSelectionParams = null;
```

Comment:
Isn't this already handled in this.removeTempHighlight()

Reply:
Yes bro, this is handled. I will remove from here bro